projects/libressl/build.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash -eux
+#
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+mkdir -p $WORK/libressl
+cd $WORK/libressl
+
+CMAKE_DEFINES=""
+if [[ $CFLAGS = *sanitize=memory* ]]
+then
+  CMAKE_DEFINES+=" -DOPENSSL_NO_ASM=1"
```

Comment:
Just a warning that if the project uses any ASM that isn't standard for amd64 it may run locally but not on the instances we uses for fuzzing.
Assuming this isn't actually the case since it isn't a problem for openssl